### ISSUE TITLE
Client example available?

### ISSUE THREAD

**Issue by wdevore** [inkyblackness/imgui-go-examples#7]
I have been trying to imgui-go package and studying the examples in this repo but something isn't adding up. The examples all require an the internal package which isn't available to external clients. Is there an example that shows a typical hello-world using the imgui-go library using glfw. Worst case is I copy the internal platform and renderer into my own project but I was hoping to avoid that or perhaps I'm missing something.

Thanks.

**Comment by dertseha** [comment]
Hello there and thank you for your message!
The examples in this repository *are* the minimal "hello world" possible. As the main library, imgui, does not come with its platform/render-specific code, it is necessary to hide the complexity behind platforms and renderers. So, the tagline is "To use imgui, implement platform/renderer like this example".

It is intentional that the implementation of the examples is hidden from external import:
* The actual rendering will probably be specialized to your solution (main reason will be the use of extra textures)
* Your use of your platform (e.g. glfw) will probably be different if you have an existing project already
* This frees up to showcase the examples with specialized, minimal implementation necessary and not consider everyone's extra cases.

In short, it is *meant* to be copied into your project, and then (highly) modified to your cases. After all, you shouldn't need to import something called `examples` into production code :)

**Comment by wdevore** [comment]
Hi @dertseha , Thanks for the reply. That is what I finally came to realize and ultimately did. ;-)
